extensions/mssql/src/objectManagement/ui/restoreDatabaseDialog.ts

```diff
@@ -108,26 +112,30 @@ export class RestoreDatabaseDialog extends ObjectManagementDialogBase<Database,
 		};
 		tabs.push(this.generalTab);
 
-		this.filesTab = {
-			title: localizedConstants.FilesSectionHeader,
-			id: this.filesTabId,
-			content: this.createGroup('', [
-				this.initializeRestoreDatabaseFilesSection(),
-				this.initializeRestoreDatabaseFilesDetailsSection()
-			], false)
-		};
-		tabs.push(this.filesTab);
+		if (!this.isManagedInstance) {
+			this.filesTab = {
+				title: localizedConstants.FilesSectionHeader,
+				id: this.filesTabId,
+				content: this.createGroup('', [
+					this.initializeRestoreDatabaseFilesSection(),
+					this.initializeRestoreDatabaseFilesDetailsSection()
+				], false)
+			};
+			tabs.push(this.filesTab);
+		}
 
-		this.optionsTab = {
-			title: localizedConstants.OptionsSectionHeader,
-			id: this.optionsTabId,
-			content: this.createGroup('', [
-				this.initializeRestoreOptionsSection(),
-				this.initializeTailLogBackupSection(),
-				this.initializeServerConnectionsSection()
-			], false)
-		};
-		tabs.push(this.optionsTab);
+		if (!this.isManagedInstance) {
```

Comment:
You can combine this with the if-case above it, since they have the same if-condition.